src/Path/Internal/Include.hs

```diff
@@ -0,0 +1,142 @@
+-- This template expects CPP definitions for:
```

Comment:
This is just copied previous Internal.hs with import of System.FilePath made dependent on PLATFORM_NAME macro.